app/backend/wells/migrations/0137_add_hydraulic_info_postgis_well_view.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.2.28 on 2022-12-03 21:19
+
+from django.db import migrations
+
+UPDATE_GWELLS_WELLS_VIEW_SQL = """
+    DROP VIEW postgis_ftw.gwells_well_view;
+    CREATE VIEW postgis_ftw.gwells_well_view AS
+        SELECT
+            well_tag_number,
+            aquifer_id,
+            artesian_conditions AS artesian,
+            COALESCE
+              (hydraulic_conductivity::text, transmissivity::text, storativity::text)
+              IS NOT NULL 
+              as has_hydraulic_info,
+            street_address,
+            identification_plate_number,
+            observation_well_number,
+            obs_well_status_code AS observation_well_status_code,
+            ems,
+            well_publication_status_code = 'Published' AS is_published,
+            geom
+        FROM well
+        WHERE geom IS NOT NULL;
+        GRANT SELECT ON postgis_ftw.gwells_well_view TO ftw_reader;
+"""
+
+class Migration(migrations.Migration):
+    """
+    This migration adds the field 'has_hydraulic_info' to the gwells_well_view
+    under the postgis_ftw schema in the gwells database. The purpose of this field
+    is to check whether there is legacy pump test data for a well, and then display
+    a different well icon on the well search map if this data exists.
+    """
+    dependencies = [
+        ('wells', '0136_add_technical_report_field'),
+    ]
+
+    operations = [
+        migrations.RunSQL(UPDATE_GWELLS_WELLS_VIEW_SQL)
```

Comment:
I think we should provide a `reverse_sql` here, just in case we need to rollback the migration.

Reply:
Added reverse migration sql code